go/vt/vtgate/evalengine/translate_builtin.go

```diff
@@ -719,7 +725,7 @@ func (ast *astCompiler) translateCallable(call sqlparser.Callable) (IR, error) {
 
 	case *sqlparser.CurTimeFuncExpr:
 		if call.Fsp > 6 {
-			return nil, vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "Too-big precision 12 specified for '%s'. Maximum is 6.", call.Name.String())
+			return nil, vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "Too-big precision %d specified for '%s'. Maximum is 6.", call.Fsp, call.Name.String())
```

Comment:
This was incorrectly hardcoding a value instead of using the one provided by the user.